lib/runner.js

```diff
@@ -606,6 +607,9 @@ Runner.prototype.run = function(fn){
   var self = this,
     rootSuite = this.suite;
 
+  // If there is an `only` filter
+  if(this.hasOnly) rootSuite = filterOnly(rootSuite);
```

Comment:
Okay, I see, you are also using`filterOnly` as a filter function. You only return `Boolean` or `suite`.
Why not change the line to:
```javascript
if(this.hasOnly) filterOnly(rootSuite);
```
You were always intending to return `rootSuite` here, right?